apps/cli/src/node/docker-compose-dev.yaml

```diff
@@ -158,15 +158,28 @@ services:
         depends_on:
             validator:
                 condition: service_healthy
+            traefik-config-generator:
+                condition: service_completed_successfully
         healthcheck:
-            test: ["CMD", "traefik", "healthcheck"]
+            test: ["CMD", "traefik", "healthcheck", "--ping"]
```

Comment:
is this necessary? it was already working, right?

```suggestion
            test: ["CMD", "traefik", "healthcheck"]
```

Reply:
I already tested it when I had enabled docker and it was not working. 

I'll test it again with new configuration.

Reply:
@endersonmaia We should define it otherwise we will get this error : 


```
dependency failed to start: container sunodo-node-proxy-1 is unhealthy
```

![image](https://github.com/sunodo/sunodo/assets/51019382/8cc01543-f6be-49fe-b958-e5be86b7d8b4)



with `--ping` all works: 

![Uploading image.png…]()